protex/update.py

```diff
@@ -314,9 +314,9 @@ def _get_positions_for_mutation_sites(self) -> tuple[dict, dict]:
             pos_list.append(
                 pos[
                     residue.get_idx_for_atom_name(
-                        self.ionic_liquid.templates.states[residue.original_name][
+                        self.ionic_liquid.templates.states[residue.current_name][
```

Comment:
Shouldn't it be current name? i.e. if it was in the beginning IM1H (=original_name=current_name => atom_name=H7), then there was a transfer and it became IM1 (=current_name => atom_name=N2 but original_name=IM1H => atom_name=H7) than a check should be made with HOAC for possible transfers, hence the nitrogen and not hydrogen atom should be checked against the hydrogen atom of HOAC.
Or am I missing something?